jerry-core/ecma/builtin-objects/ecma-builtin-json.c

```diff
@@ -1253,15 +1253,24 @@ ecma_builtin_json_serialize_property (ecma_json_stringify_context_t *context_p,
   }
 
   /* 3. */
-  if (ecma_is_value_object (value))
+  if (ecma_is_value_object (value) || ecma_is_value_bigint (value))
   {
-    ecma_object_t *value_obj_p = ecma_get_object_from_value (value);
+    ecma_value_t temp_val = ecma_op_to_object (value);
```

Comment:
I don't like the name of `temp_val`. What about `to_object_value` ? And it would be good to avoid this conversion for objects (which is a gain for ES5.1)